R/pkg/R/mllib.R

```diff
@@ -896,9 +898,10 @@ setMethod("summary", signature(object = "LogisticRegressionModel"),
 #' summary(savedModel)
 #' }
 #' @note spark.mlp since 2.1.0
```

Comment:
we are targeting 2.1.0 for this change yes? otherwise it is a breaking signature change